addon/globalPlugins/nao.py

```diff
@@ -32,13 +34,25 @@
 pdfToPngToolPath = "\""+os.path.join (addonPath, "tools", "pdftopng.exe")+"\""
 pdfToImagePath = "" + os.path.join (addonPath, "images") + ""
 pdfToImageFileNamePath = pdfToImagePath + "\\img"
+ADDON_SUMMARY = addonHandler.getCodeAddon().manifest["summary"]
 
 class GlobalPlugin(globalPluginHandler.GlobalPlugin):
+	try:
+		scriptCategory = unicode(ADDON_SUMMARY)
+	except NameError:
+		scriptCategory = str(ADDON_SUMMARY)
```

Comment:
In which case is `str(...)` required? I have tried with an UTF8-encoded manifest and a value with an accent ("à") in the summary. And the string is encoded correctly in the input gesture dialog.